FirebaseRemoteConfigSwift.podspec

```diff
@@ -21,9 +21,9 @@ app update.
 
   s.swift_version           = '5.3'
 
-  ios_deployment_target = '10.0'
+  ios_deployment_target = '11.0'
```

Comment:
Why is the minimum OS version change needed?  This makes the change breaking and would require us to wait for the next major Firebase release.

Reply:
@paulb777 This change is issued by SwiftUI problems to build with armv7 devices, same as https://github.com/firebase/firebase-ios-sdk/issues/8754.
I fix it with macro for build architecture in https://github.com/firebase/firebase-ios-sdk/pull/9918/commits/aadf50aa3eabeedb6264cc8a1776c84927bbda76.